manifests/virt-handler.yaml.in

```diff
@@ -9,6 +9,7 @@ spec:
       labels:
         daemon: virt-handler
     spec:
+      serviceAccountName: kubevirt-infra
```

Comment:
virt-handler should have a reduced role that limits it to vm objects with verbs the prevent it from being able to delete anything.